.github/workflows/ci.yml

```
@@ -9,8 +9,8 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-        python-version: [3.9, 3.10]
+        os: [ubuntu-latest, macos-latest]
+        python-version: ["3.9", "3.10"]
```

```suggestion
        python-version: ["3.11", "3.12"]
```
Seems macos-latest doesn't have the old versions available.  New should be fine, I think.  Let's see if it works.